tools/spark.md

```diff
@@ -231,44 +231,47 @@ The MozTelemetry Library
 ------------------------
 
 We have provided a library that gives easy access to the raw telemetry ping data.
-For example usage, see the 
-[Telemetry Hello World](http://reports.telemetry.mozilla.org/post/tutorials/telemetry_hello_world.kp)
+For example usage, see the
+[Telemetry Hello World](https://reports.telemetry.mozilla.org/post/tutorials/telemetry_hello_world.kp)
 example notebook.
-Detailed documentation for the library can be found at the 
-[Python MozTelemetry Documentation](http://python-moztelemetry.readthedocs.io).
+Detailed documentation for the library can be found at the
+[Python MozTelemetry Documentation](https://python-moztelemetry.readthedocs.io).
 
 ### Using the Raw Ping Data
 
-First off, import the moztelemetry library using the following:
+First off, import the `moztelemetry` library using the following:
 
-`from moztelemetry import get_pings`
+`from moztelemetry.dataset import Dataset`
 
-If you need any other functions, just comma separate them with
-`get_pings`.
 
 The ping data is an RDD of JSON elements. For example, using the
 following:
 
 ```python
-pings = get_pings(sc, app="Firefox", channel="nightly", build_id=("20160901000000", "20160901999999"), fraction=0.01)
+pings = Dataset.from_source("telemetry") \
+    .where(docType='main') \
+    .where(submissionDate="20180101") \
+    .where(appUpdateChannel="nightly") \
+    .records(sc, sample=0.01)
 ```
 
-returns an RDD of 1/100th of Firefox Nightly JSON pings for all builds
-from September 1 2016. Now, because it's JSON, pings are easy to access.
+returns an RDD of 1/100th of Firefox Nightly JSON pings submitted on
```

Comment:
I believe this returns a DataFrame not an RDD.

Reply:
Nope, it's still an RDD :(